lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/audio_output.rb

```diff
@@ -0,0 +1,74 @@
+# -*- coding: binary -*-
+require 'rex/post/meterpreter'
+require 'bindata'
+
+module Rex
+  module Post
+    module Meterpreter
+      module Ui
```

Comment:
Minor: remove indention to be more consistent with other stdapi classes. For example, see `lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb`.